pkg/wshrpc/wshremote/wshremote.go

```
@@ -348,11 +348,113 @@ func (impl *ServerImpl) RemoteFileCopyCommand(ctx context.Context, data wshrpc.C
+					}
+					defer file.Close()
+				}
+				_, err = copyFileFunc(path, info, file)
+				return err
+			})
+			if err != nil {
+				return fmt.Errorf("cannot copy %q to %q: %w", srcUri, destUri, err)
+			}
+		} else {
+			file, err := os.Open(srcPathCleaned)
+			defer file.Close()
```

golangci-lint (1.62.2)

SA5001: should check error returned from os.Open() before deferring file.Close()

(staticcheck)